src/ip/Intercept.h

```diff
@@ -30,8 +30,8 @@ class Intercept
     Intercept() : transparentActive_(0), interceptActive_(0) {}
     ~Intercept() {};
 
-    /** Perform NAT lookups */
-    bool Lookup(const Comm::ConnectionPointer &newConn, const Comm::ConnectionPointer &listenConn);
+    /// perform NAT lookups
+    bool LookupNat(const Comm::ConnectionPointer &newConn);
```

Comment:
Let's try to reduce the "foo is foo" effect. Please adjust the wording to match reality -- NAT is not my area of expertise.

```suggestion
    /// use NAT to set the source/destination addresses of the given freshly accepted connection
    bool lookupNat(Comm::Connection &);
```

The above also changes the API to guarantee Connection object existence. Please use a local `newConn` variable to keep the method diff focused on true changes inside LookupNat()...